onnx/reference/ops/op_quantize_linear.py

```diff
@@ -142,6 +142,7 @@ def _run(  # noqa: PLR0911
         saturate: bool = True,
         block_size: int | None = None,
         output_dtype: np.dtype | None = None,
+        precision: np.dtype | None = None,
```

Comment:
As above ... this is really an integer corresponding to onnx datatype enumeration, right? Shouldn't we map that to np.dtype internally in the implementation?

Reply:
Looking at `op_cast.py` the `to` attribute is of TensorProto type. 
But the existing attribute of `output_dtype` here takes np.dtype. This is probably a mistake?

Reply:
I think so. We should be passing in the same values as specified in the ONNX spec (for integer value attributes).

Reply:
Updated.